test/integration/connect/envoy/case-wanfed-gw/primary/verify.bats

```diff
@@ -10,6 +10,18 @@ load helpers
   assert_upstream_has_endpoints_in_status 127.0.0.1:19000 secondary HEALTHY 1
 }
 
+@test "gateway-primary should have healthy endpoints for secondary servers" {
```

Comment:
Adding more explicit line-item steps here to aid in future stuck state debugging.